### PR TITLE
ENH: Add CircleCI 2.0 configuration

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+              -DBUILDNAME:STRING=External-ITKAnisotropicDiffusionLBR-${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM} \
+                ~/ITKAnisotropicDiffusionLBR
+      - run:
+          name: Build and Test
+          command: |
+            ctest -VV -D Experimental
+
+workflows:
+    version: 2
+    build-test-package:
```
Comment (jcfr) [inline .circleci/config.yml]: Look like the packaging step is missing ?

Comment (thewtex) [reply .circleci/config.yml]: Should have marked it as WIP -- should be about ready, now.

**Comment by jcfr** [comment]
While CircleCI reports:

```
Uploading /home/circleci/ITKAnisotropicDiffusionLBR/dist to s3://circle-production-customer-artifacts/picard/55b99b23f7d0bc181284e51e/59a4e9fc17a8e10001168678-0-build/artifacts/dist
Uploaded /home/circleci/ITKAnisotropicDiffusionLBR/dist/itk_anisotropicdiffusionlbr-0.1.0-cp27-cp27m-manylinux1_x86_64.whl
Uploaded /home/circleci/ITKAnisotropicDiffusionLBR/dist/itk_anisotropicdiffusionlbr-0.1.0-cp27-cp27mu-manylinux1_x86_64.whl
Uploaded /home/circleci/ITKAnisotropicDiffusionLBR/dist/itk_anisotropicdiffusionlbr-0.1.0-cp34-cp34m-manylinux1_x86_64.whl
Uploaded /home/circleci/ITKAnisotropicDiffusionLBR/dist/itk_anisotropicdiffusionlbr-0.1.0-cp35-cp35m-manylinux1_x86_64.whl
Uploaded /home/circleci/ITKAnisotropicDiffusionLBR/dist/itk_anisotropicdiffusionlbr-0.1.0-cp36-cp36m-manylinux1_x86_64.whl
```

I couldn't find where to access the wheels ... may be it is related to current CircleCI outage ?

**Comment by thewtex** [comment]
Package artifacts are [available in the Artifacts tab](https://circleci.com/gh/InsightSoftwareConsortium/ITKAnisotropicDiffusionLBR/37#artifacts/containers/0)

**Pull request merged**